README.md

```diff
@@ -166,6 +105,44 @@ fn rust_ext(_py: Python<'_>, m: &PyModule) -> PyResult<()> {
 }
 ```
 
+## Requirements
+- Rust >= 1.41.1
+  - Basically, our MSRV follows the one of [PyO3](https://github.com/PyO3/pyo3)
+- Python >= 3.6
+  - Python 3.5 support is dropped from 0.13
+- Some Rust libraries
+  - [ndarray](https://github.com/rust-ndarray/ndarray) for Rust-side matrix library
+  - [PyO3](https://github.com/PyO3/pyo3) for Python bindings
+  - And more (see [Cargo.toml](Cargo.toml))
+- [numpy](https://numpy.org/) installed in your Python environments (e.g., via `pip install numpy`)
+  - We recommend `numpy >= 1.16.0`, though older versions may work
```

Comment:
TBH I think having `requirements` at the top is probably better? That way it's super easy for users to see them (and obvious if they can't meet the requirements), and it's not such a long section that readers won't see the example underneath.

Reply:
Yes, I was rather unsure about this one. I wanted to keep it together with the section on ndarray, but let's split them up since that one is the main issue here.